sdv/sequential/par.py

```diff
@@ -217,3 +221,112 @@ def _fit(self, processed_data):
 
         LOGGER.debug(f'Fitting {self.__class__.__name__} model to table')
         self._fit_sequence_columns(processed_data)
+
+    def _sample_from_par(self, context, sequence_length=None):
+        """Sample new sequences.
+
+        Args:
+            context (pandas.DataFrame):
+                Context values to use when generating the sequences.
+            sequence_length (int):
+                Length of each sequence to sample. If not
+                given, the sequence length will be sampled from
+                the model.
+
+        Returns:
+            pandas.DataFrame:
+                Table containing the sampled sequences in the same
+                format as that he training data had.
+        """
+        # Set the sequence_key as index to properly iterate over them
+        if self._sequence_key:
+            context = context.set_index(self._sequence_key)
+            # reorder context columns
+            context = context[self.context_columns]
+
+        should_disable = not self._model_kwargs['verbose']
+        iterator = tqdm.tqdm(context.iterrows(), disable=should_disable, total=len(context))
+
+        output = []
+        for sequence_key_values, context_values in iterator:
+            context_values = context_values.tolist()
+            sequence = self._model.sample_sequence(context_values, sequence_length)
+            if self._sequence_index:
+                sequence_index_idx = self._data_columns.index(self._sequence_index)
+                diffs = sequence[sequence_index_idx]
+                start = sequence.pop(-1)
+                sequence[sequence_index_idx] = np.cumsum(diffs) - diffs[0] + start
+
+            # Reformat as a DataFrame
+            sequence_df = pd.DataFrame(
+                dict(zip(self._data_columns, sequence)),
+                columns=self._data_columns
+            )
+            sequence_df[self._sequence_key] = sequence_key_values
+            for column, value in zip(self.context_columns, context_values):
+                sequence_df[column] = value
+
+            output.append(sequence_df)
+
+        output = pd.concat(output)
+        output = output[self._output_columns].reset_index(drop=True)
+        if self._sequence_index:
+            output = output.rename(columns={
+                self._sequence_index: self._sequence_index + '.value'
```

Comment:
Will this `.value` go away after RDT changes?

Reply:
yes it will!